governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+
+Third, the proposed Standing Order concerns revenue return and thus, the technical means to achieve this is an important question for Maker governance. But Emilia's Forum post offers scant insight into the technical details and the few words she does offer are vague. Which conduit contract, for instance? (See [here](http://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384/2) for an example of why it is important to be clear on this point.) 
+
+Fourth, the title of Emilia's post ("Standing Order Change") is misleading and likely to cause confusion. This is because two standing orders are implicated in her post. Though only one is being currently proposed for a Governance poll, the other one is the subject of a Forum poll. The presence of the Forum poll may easily mislead readers to believe that the latter standing order is referenced in the title. 
+
+### Forum Polling
+
+First, Emilia's deployment of a Forum poll under these circumstances is confusing. It's unclear whether she deployed a Forum poll to take on the guise of a MIP24 Signal Request. (See discussion above.) 
+
+The structure/content of the poll also fails on a few fronts. There's no introduction to, or summary of, the essential context that participants would need prior to meaningful participation in the poll. There's no Abstain option, which can be a valuable signal for voters who wish to express ambivalence or proactively protest. In each poll option, two different values/scenarios are combined, making any "signal" impossible to discern. The fourth poll option of ''Something Else" is similarly useless. Finally, the title of Emilia's post, "Standing Order Change", is likely to cause significant misunderstanding: voters may believe that the poll (on the logic of the future ALM order) represents the "standing order change" referenced in the title. 
+
+The biggest problem: Emilia's post may lead the community to believe that the poll outcomes have a determinative say in the logic of the replacement ALM order. If Emilia wishes to gauge community sentiment and is yet aware of her mandate as an Arranger, then she would have to acknowledge that deciding the logic of the ALM transfer orders ultimately lies within the purview of domain experts. 
+
+### Re: Faraz's Reply
+
+Faraz's reply doesn't help matters. There is his use of the singular "this", in conjunction with the plural "changes". Is this a typo? It's not clear what he's approving, either. There are two Standing Orders at issue. The first is the proposed Order relating to revenue return, which has obvious ambiguities. The second is the current Order relating to ALM transfers which Emilia improperly "invalidated" in her Forum post. It is possible that Faraz, in issuing such a confusing reply and approving either/both Order(s), has not fulfilled his mandate as a Facilitator. 
```

Comment:
"Faraz's reply doesn't help matters. There is his use of the singular "this", in conjunction with the plural "changes". Is this a typo?" - Well its definitely a typo, the question is whether the fix to the typo results in singular or plural approval. I feel like you've covered this. 

"It is possible that Faraz, in issuing such a confusing reply and approving either/both Order(s), has not fulfilled his mandate as a Facilitator." - Yeah agree.

I believe I raised the point with you previously about the facilitators domains, and not stepping outside governance in most situations. This is a case I would consider a valid exception. Faraz has failed to comment on the removal of a key piece of stability infrastructure, which borders on gross negligence. At the least, that warrants approaching them privately, and asking them what they hell they are playing at.